precode.sh

```diff
@@ -7,37 +7,56 @@
 #   dir3
 #       dir4
 
+mkdir -p task/dir1 task/dir2 task/dir3 
```

Comment:
При использовании флага `-p` можно сразу так делать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```